### PR TITLE
Add output file names to HTML + misc

### PR DESCRIPTION
**Pull request by RossBoylan**
Each commit deals with a distinct issue.  In order:
1. Fix a missing `;` that triggered some errors/warnings.
2. Add filename to the codefence rendered in HTML--if one was specified
3. A bit more instruction on developing code.  As noted in that commit, I would have loved to explain how the instructions relate to what happens automatically with `F5`, but I'm not sure what that is...

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 * Ensure `yarn` is installed: `npm install -g yarn`
 * Ensure `typescript` compiler is installed: `npm install -g typescript`
 * Ensure dependencies are installed: `yarn`
+* Change code in the `.literate` files
+* Rebuild with the `VSCode` command `Literate: Process` (`literate.process`)
```
Comment (jesterKing) [inline CONTRIBUTING.md:7]: This step is triggered automatically on any file change in a workspace. It is not necessary to run the `literate.process` command.

Also just opening a workspace with literate files will trigger the command.

If you could remove this step.

File: src/extension.ts [hunk 2]
```diff
@@ -1,1430 +1,1431 @@
...
+        let add = match.groups.add;
+        let fileName = match.groups.fileName;
+        if (name) {
+          root = root || '';
+          add = add || '';
+          fileName = fileName || '';
+          rendered = protectFragmentTags(rendered);
+          rendered =
+`<div class="codefragment">
+<div class="fragmentname">&lt;&lt;${name}&gt;&gt;${root}${add} ${fileName}</div>
```
Comment (jesterKing) [inline src/extension.ts:357]: I was debating with myself early on whether I wanted to show this or not. I went along with what PBR book does, and that is to not show the filenames.

For now I think it is ok to show, and introduce later an extension setting that can control the showing of filenames in rendered output.

**Review by jesterKing — CHANGES_REQUESTED** [review]
Thank you for your PR!

One small change request for the CONTRIBUTING.md.

And could you rebase on latest jesterKing:main, please?

File: CONTRIBUTING.md (hunk 1 above)
Comment (jesterKing) [reply CONTRIBUTING.md:7]: If you think it is important to educate users about the existance of the command then perhaps a rewording of the step to something like:

If the code changes in `.literate` files don't automatically rebuild the HTML and designated source files you can run the `VSCode` command `Literate: Process` (`literate.process`).

**Comment by jesterKing** [comment]
Also, @RossBoylan, please add your name to the CONTRIBUTORS.md file in this PR :)

**Comment by RossBoylan** [comment]
Will try again with new branch.

**Pull request closed without merging**